docs/en/04_Changelogs/4.0.0.md

```diff
@@ -672,30 +672,42 @@ Those wishing to customise the CMS should read about how to
 
 All static files (images, javascript, stylesheets, fonts) used for the CMS and forms interfaces
 in `framework` and `cms` have moved locations. These assets are now placed in a `client/` subfolder,
-to account for a structural change where both javascript and styles are co-located in component-specific folders.
+which are also sub-folders of the modules which now reside in `vendor`.
+
 This will affect you if you have used `Requirements::block()` on files in the `framework/` or `cms/` folder.
+
+In order to identify resources it is preferred to use the new module-root prefixed string form when
+adding requirements.
```

Comment:
We explain elsewhere that modules are now stored deterministically, but do we need to explain how that is computed? I feel like we're missing a reference to `composer.json` somewhere. I'm not sure I would intuit straight away where `silverstripe/admin: ` came from if I was new to SS.

Reply:
I'll mention it here.